user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -45,14 +45,34 @@ public UserServiceImpl(PersistentEntityRegistry registry, ActorSystem system, Ma
     }
 
     @Override
+    public ServiceCall<Auth, Auth> updateAuth() {
+        return auth -> {
+            return authRef(auth.getUsername()).ask(new AuthCommand.UpdateAuth(auth.getId(), auth.getUsername(), auth.getPassword()));
```

Comment:
why return the `Auth`?

It's an object that contains a clear-text password and we should limit the password exposure.
I think a `Done` is more than enough.